src/main/java/amazonreviewsapp/springboot/model/User.java

```diff
@@ -0,0 +1,74 @@
+package amazonreviewsapp.springboot.model;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.Set;
+
+@Entity
+@Table(name = "users")
+public class User {
```

Comment:
entity creating should be in a separate PR.

Reply:
I thought Bohdan said that we can join several prs, anyway, what's the difference?